### ISSUE TITLE
Thrown SHADE at the N64 sceen

### ISSUE THREAD

**Issue by theboy181** [gonetz/GLideN64#1866]

**Comment by gonetz** [comment]
Impressive article, written by a person engrossed in N64 emulation.
However, it is confusing to see it in open issues list.
Bad that we does not have a billboard here.

**Comment by oddMLan** [comment]
We shared our opinions about it on the Discord server. It would be nice if you and @olivieryuyu showed up more often! ☺️

**Comment by olivieryuyu** [comment]
Interesting article, I do not agree on everything but it is a good one.

I am not really a fan of Discord (i don think I even remember how to access it). For info I slowed down the pace of the ucode deciphering (Indy was exhausting). I am currently working on S2DEX. I am now focusing on the background commands, with hope to imrprove quality in some games. It is a hard one. I am not surprised that current implementation is not accurate.

**Comment by theboy181** [comment]
@olivieryuyu there are loads of developers on Discord64. Please take some time every once in a while. there might be some people that can help. 

There are also quite a few people who love what you guys are doing, that would just love to be surrounded by more great developers. 

PS sorry for posting this article here, I just don't know how else to share with you guys outside of discord and the old GLideN64 forum. (which is inactive)

I wanted to let you guys know that I was willing to support ht project again, once you start focus on the LLE side of things. 

Door is always open.
https://discord.gg/r4eSFZ2

**Comment by olivieryuyu** [comment]
The article can be postes on the wiki page. 

There is few things to get fixed in hle. Then indeed fb effects and rdp fixes will have to be improved where necessary.

**Comment by olivieryuyu** [comment]
@theboy181 

can you move this article to the wiki?

**Comment by theboy181** [comment]
Yeah. This was the only way I know how to reach you guys.  Please cameo on discord64 more often. ;)
